xclim/cli.py

```diff
@@ -0,0 +1,239 @@
+# -*- coding: utf-8 -*-
+"""
+xclim command line interface module
+"""
+import inspect
+
+import click
+import xarray as xr
+from dask.diagnostics import ProgressBar
+
+import xclim as xc
+from xclim import atmos
+from xclim import land
+from xclim import seaIce
```

Comment:
Just an FYI, but it's no longer necessary to import these individual modules, just `ensembles` and `subset` need explicit import.